raw.go

```diff
@@ -95,6 +95,12 @@ func (c *Conn) SetBPF(filter []bpf.RawInstruction) error {
 	return c.p.SetBPF(filter)
 }
 
+// SetPromisc enables/disables the interface promiscuous mode
+// 0 == disable, 1 == enable
+func (c *Conn) SetPromisc(m int) error {
```

Comment:
Bit new to Go, does this look like an acceptable way to supply bool to the function but change to integer for the syscalls?

```
func (p *packetConn) SetPromiscuous(b bool) error {

	m := 1
	if !b {
		m := 0
	}

	return syscall.SetBpfPromisc(p.fd, m)
}
```